apps/prosody/vars/all/main.yml

```diff
@@ -0,0 +1,95 @@
+# -- route
+prosody_host: "prosody.{{ namespace_name }}.{{ domain_name }}"
+
+# -- Prosody
+
+prosody_image_name: "fundocker/prosody"
+prosody_image_tag: "latest"
+prosody_general_admins: []
+prosody_lua_libevent: true
+prosody_cross_domain_bosh: true
+prosody_consider_bosh_secure: true
+prosody_general_modules_enabled:
+  - roster
+  - saslauth
+  - dialback
+  - disco
+  - carbons
+  - pep
+  - private
+  - blocklist
+  - vcard4
+  - vcard_legacy
+  - version
+  - uptime
+  - time
+  - ping
+  - register
+  - admin_adhoc
+prosody_general_modules_disabled: []
+prosody_allow_registration: false
+prosody_c2s_require_encryption: false
+prosody_allow_unencrypted_plain_auth: true
+prosody_s2s_require_encryption: false
+prosody_s2s_secure_auth: false
+prosody_s2s_insecure_domains: []
+prosody_s2s_secure_domains: []
+prosody_general_authentication: "internal_hashed"
+prosody_storage: "internal"
+prosody_archive_expires_after: "1w"
+prosody_log_level: "info"
+prosody_vertificates_directory: "certs"
+prosody_asap_accepted_issuers:
+  - "prosody"
+prosody_asap_accepted_audiences: 
+  - "prosody"
+prosody_client_connection_port: 5222
+prosody_nodeport_client_connection_port: 30007
+prosody_http_port: 5280
+prosody_external_components_port: 5347
+prosody_nodeport_external_components_port: 30008
+prosody_secret_name: "prosody-{{ prosody_vault_checksum | default('undefined_prosody_vault_checksum') }}"
+
+# -- Volumes
+# The storage volume is used only when the prosody storage is internal.
+prosody_storage_volume_size: 2Gi
```

Comment:
It seems to be huge to store a plain text files :sweat_smile:

Reply:
Indeed 🙂